### ISSUE TITLE
[support] adding markdown-it html options

### ISSUE THREAD

**Issue by chasebank** [evanfuture/vue-yes-blog#1]
Hey, sorry for posting this here, but I think I've got a good shot getting a response from you and the info may help others.

I'm trying to add markdown-it options for rendering html in the md files.

The standard [markdown-it options](https://github.com/markdown-it/markdown-it#init-with-presets-and-options) seem pretty straight forward, but I think I'm getting hung up on how to add that to the webpack config, especially with running the additional json loader on the same files...

Here's what I've tried so far, to no avail:

```javascript
}
  test: /\.md/,
  // loaders: ['json-loader', 'markdown-it-front-matter-loader'],
  // options: {
  //   html: true,
  //   linkify: true,
  //   typographer: true
  // }
  // loaders: [{
  //   loader: 'json-loader'
  // }, {
  //   loader: 'markdown-it-front-matter-loader',
  //   options: {
  //     html: true,
  //     linkify: true,
  //     typographer: true
  //   }
  // }]
  use: [{
    loader: 'json-loader'
  }, {
    loader: 'markdown-it-front-matter-loader',
    options: {
      html: true,
      linkify: true,
      typographer: true
    }
  }]
}
```

any ideas? Great article on CSS-tricks btw

**Comment by evanfuture** [comment]
Hey @chasebank, sorry to let you down there and not respond for 6 days.  It's been a crazy week.  Thanks for the kind words about the article.

It looks like you can provide a 'markdown-it-front-matter' key/value pair as shown in the readme here: https://www.npmjs.com/package/markdown-it-front-matter-loader

I wouldn't be sure of where to place that without trying it out myself (it seems like the loader-syntax being used there is outdated), but it doesn't look like it's impossible!

Let me know how you get on.

**Comment by chasebank** [comment]
Thanks for the reply Evan, no worries on the delay!

Good observation on the outdated syntax. I think that may be part of the issue. I got some vulgar webpack errors saying something about things changed with version 2.

At any rate, I eventually switched to [markdown-with-front-matter-loader](https://github.com/matthewwithanm/markdown-with-front-matter-loader) and that seems to have done the trick. At first I was getting errors that I narrowed down to the json-loader. It looks like this markdown loader eliminates the need for the json one entirely. And it compiles html out of the box, without the need for extra options. For better or worse...

So, problem solved!
